patches/git/0001-Initial-Managarm-support.patch

```diff
@@ -0,0 +1,35 @@
+From 53584e9beb2c607d6dd77422db652c6014ddabcb Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Fri, 16 Jul 2021 15:58:28 +0200
+Subject: [PATCH] Initial Managarm support
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
+ config.mak.uname  | 12 ++++++------
+ 1 file changed, 6 insertions(+), 6 deletions(-)
+
+diff --git a/config.mak.uname b/config.mak.uname
+index cb443b4e02..6cb9520c54 100644
+--- a/config.mak.uname
++++ b/config.mak.uname
+@@ -4,12 +4,12 @@
+ # Microsoft's Safe Exception Handling in libraries (such as zlib).
+ # Typically required for VS2013+/32-bit compilation on Vista+ versions.
+ 
+-uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+-uname_M := $(shell sh -c 'uname -m 2>/dev/null || echo not')
+-uname_O := $(shell sh -c 'uname -o 2>/dev/null || echo not')
+-uname_R := $(shell sh -c 'uname -r 2>/dev/null || echo not')
+-uname_P := $(shell sh -c 'uname -p 2>/dev/null || echo not')
+-uname_V := $(shell sh -c 'uname -v 2>/dev/null || echo not')
++uname_S := #$(shell sh -c 'uname -s 2>/dev/null || echo not')
++uname_M := #$(shell sh -c 'uname -m 2>/dev/null || echo not')
++uname_O := #$(shell sh -c 'uname -o 2>/dev/null || echo not')
++uname_R := #$(shell sh -c 'uname -r 2>/dev/null || echo not')
++uname_P := #$(shell sh -c 'uname -p 2>/dev/null || echo not')
++uname_V := #$(shell sh -c 'uname -v 2>/dev/null || echo not')
```

Comment:
Why is this done?

Reply:
If left as it was, the uname will match Linux, and we don’t build for Linux. In my memory serenity used to patch it like this (and they did), but they removed this patch a few months ago.

Reply:
Does this run on the host? Where does the uname show up when running git?

Reply:
Serenity reinstated the patch not too long ago, see [this issue](https://github.com/SerenityOS/serenity/issues/9357) and the [linked pr](https://github.com/SerenityOS/serenity/pull/9377) for an explanation why.

Reply:
these can be overwritten on the make command line regardless of the state of the makefile